src/app/news/news-list-item.component.ts

```diff
@@ -131,4 +134,12 @@ export class NewsListItemComponent implements OnChanges, AfterViewChecked {
       (!this.targetLocalPlanet || (!this.newsService.postSharedWithCommunity(this.item) && this.isMainPostShared));
   }
 
+  shouldShowIndication() {
+    if (this.shareTarget === this.planetCode) {
+      return this.activeMembers.user.findIndex(this.item.doc.user._id) !== -1 &&
```

Comment:
This isn't quite it.  You don't need to look at the `loginActivities`, those are records of each login for the members of the community.

You need to check each user and see if the `_id` and the `joinDate` match the user on the news document.